bitwarden_license/bit-web/src/app/secrets-manager/shared/org-suspended.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from "@angular/core";
+import { ActivatedRoute } from "@angular/router";
+import { concatMap, Subject, takeUntil } from "rxjs";
+
+import { OrganizationService } from "@bitwarden/common/admin-console/abstractions/organization/organization.service.abstraction";
+import { Icon, Icons } from "@bitwarden/components";
+
+@Component({
+  templateUrl: "./org-suspended.component.html",
+})
+export class OrgSuspendedComponent implements OnInit {
+  constructor(private organizationService: OrganizationService, private route: ActivatedRoute) {}
+
+  protected organizationName: string;
+  protected organizationId: string;
+  protected NoAccess: Icon = Icons.NoAccess;
+  private destroy$ = new Subject<void>();
+
+  async ngOnInit() {
+    this.route.params
+      .pipe(
+        concatMap(async (params) => {
+          this.organizationId = params.organizationId;
+          this.organizationName = await this.organizationService.get(this.organizationId)?.name;
+        }),
+        takeUntil(this.destroy$)
+      )
+      .subscribe();
+
+    this.organizationName = this.organizationService.get(this.organizationId)?.name;
+  }
```

Comment:
```suggestion
export class OrgSuspendedComponent implements OnInit, OnDestroy {
  constructor(private organizationService: OrganizationService, private route: ActivatedRoute) {}

  protected organizationName: string;
  protected NoAccess: Icon = Icons.NoAccess;
  private destroy$ = new Subject<void>();

  ngOnInit() {
    this.route.params.pipe(takeUntil(this.destroy$)).subscribe((params) => {
      this.organizationName = this.organizationService.get(params.organizationId)?.name;
    });
  }
```

It looks like we are setting `this.organizationName` on line 24 and 30.
 
I don't think our `ngOnInit` needs to be async because `this.organizationService.get` isn't async. 

Need to add implements `OnDestroy` onto the class.